service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -853,6 +854,19 @@ public Map<String, String> getCredentialConfig(
         storageInfo.get());
   }
 
+  @Override
+  public Map<String, String> getVendedCredentialConfig(TableIdentifier tableIdentifier) {
+    Map<String, String> vendedCredentialConfig = new HashMap<>();
+    String credentialsEndpoint =
+        String.format(
+            "/v1/%s/namespaces/%s/tables/%s/credentials",
+            catalogName, tableIdentifier.namespace().toString(), tableIdentifier.name());
```

Comment:
Does this do proper escaping of special chars in the URI path?

Cf. `IcebergCatalogAdapter.decodeNamespace()`